src/Microsoft.AspNet.Mvc.ModelBinding/Binders/MutableObjectModelBinder.cs

```diff
@@ -273,14 +273,20 @@ protected virtual void EnsureModel(ModelBindingContext bindingContext)
 
         protected virtual IEnumerable<ModelMetadata> GetMetadataForProperties(ModelBindingContext bindingContext)
         {
-            var validationInfo = GetPropertyValidationInfo(bindingContext);
-            var newPropertyFilter = GetPropertyFilter();
-            return bindingContext.ModelMetadata.Properties
-                                 .Where(propertyMetadata =>
-                                    newPropertyFilter(bindingContext, propertyMetadata.PropertyName) &&
-                                    (validationInfo.RequiredProperties.Contains(propertyMetadata.PropertyName) ||
-                                    !validationInfo.SkipProperties.Contains(propertyMetadata.PropertyName)) &&
-                                    CanUpdateProperty(propertyMetadata));
+            var propertyBindingInfo = GetPropertyBindingInfo(bindingContext);
+
+            var propertyFilter = GetPropertyFilter();
+            return 
+                bindingContext.ModelMetadata
+                .Properties
+                .Where(propertyMetadata =>
+                {
+                    return 
+                        propertyFilter(bindingContext, propertyMetadata.PropertyName) &&
+                        (propertyBindingInfo.RequiredProperties.Contains(propertyMetadata.PropertyName) ||
+                        !propertyBindingInfo.SkipProperties.Contains(propertyMetadata.PropertyName)) &&
+                        CanUpdateProperty(propertyMetadata);
+                });
```

Comment:
I had to rename some of these things, and this formatting was driving me nuts. Still don't like it so much.